ansible_runner/runner_config.py

```diff
@@ -45,12 +45,14 @@ def prepare_env(self):
         self.expect_passwords[pexpect.TIMEOUT] = None
         self.expect_passwords[pexpect.EOF] = None
 
+        # seed env with existing shell env
+        self.env = os.environ.copy()
```

Comment:
move to inside the context manager in the `try...except` block

Reply:
done